### PR TITLE
Fail on error

### PR DESCRIPTION
**Pull request by aaomidi**
We should probably just stop the build when this error happens, thoughts?

### CODE REVIEW

**Comment by aaomidi** [comment]
Alternatively we could change it here: https://github.com/microsoft/azuredatastudio/pull/9429/files#diff-00ab3d500e4e3252c8906ccfa2bcc6d2L101

**Comment by alanrenmsft** [comment]
the change looks good, please modify the test code to make it throw an error so that we can test it. after that you can revert the seeded error.

File: build/azure-pipelines/win32/sql-product-test-win32.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ steps:
     $AppProductJson = Get-Content -Raw -Path "$AppRoot\resources\app\product.json" | ConvertFrom-Json
     $AppNameShort = $AppProductJson.nameShort
     exec { $env:INTEGRATION_TEST_ELECTRON_PATH = "$AppRoot\$AppNameShort.exe"; $env:INTEGRATION_TEST_CLI_PATH = "$AppRoot\bin\$AppNameShort"; .\scripts\sql-test-integration.bat }
-  continueOnError: true
+  continueOnError: false
```
Comment (anthonydresser) [inline build/azure-pipelines/win32/sql-product-test-win32.yml]: I would just delete the line since the default behavior is to fail on error.

**Comment by corivera** [comment]
@aaomidi Would it be better to remove continueOnError from the publish step instead? We already fail that task if tests fail, plus we still get published test results on failure.

**Comment by aaomidi** [comment]
No. That step would then fail if we're running (and continuing on error) unstable tests in product build.

**Pull request merged**